### PR TITLE
Use specific RGW endpoint from zone for MDR deployment

### PR DESCRIPTION
**Pull request by petr-balogh**
Ticket:
https://issues.redhat.com/browse/OCSQE-913

### CODE REVIEW

**Comment by petr-balogh** [comment]
Tested against regressions here that I didn't break old deployment without zone:
https://url.corp.redhat.com/eb723fd

The run failed but on different thing becasue previous execution didn't cleanup properly.

**Review by clacroix12 — APPROVED** [review]

File: ocs_ci/deployment/helpers/external_cluster_helpers.py [hunk 1]
```diff
@@ -378,7 +383,12 @@ def get_rgw_endpoint():
 
     """
     rgw_endpoint = None
+    zone = config.ENV_DATA.get("zone")
```
Comment (shylesh) [inline ocs_ci/deployment/helpers/external_cluster_helpers.py:386]: @petr-balogh , I am not able to visualize about the source of this zone info. the data center zone info from config.EXTERNAL_MODE["external_cluster_node_roles"] comes from the external cluster yaml that we will have but I was wondering from where this ENV_DATA.zone gets populated? I tried dumping node info from a vmware cluster and I don't see any, if we are passing this zone info from a config file through --ocsci-conf, which file is that ? Am I missing something totally!

Comment (petr-balogh) [reply ocs_ci/deployment/helpers/external_cluster_helpers.py:386]: This is populated from multicluster job from this place: https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/-/blob/master/jobs/pipelines/qe_dr_setup/Jenkinsfile#L46 where we are creating the yaml text config.

This we looked at with @dahorak and should be created for each cluster.

Comment (petr-balogh) [reply ocs_ci/deployment/helpers/external_cluster_helpers.py:386]: But for non mdr job it's not needed and I don't want to break it.

Comment (shylesh) [reply ocs_ci/deployment/helpers/external_cluster_helpers.py:386]: @petr-balogh , got it, my mistake, I forgot that this is also related to single cluster as well.

**Review by shylesh — APPROVED** [review]
LGTM

**Pull request merged**